### PR TITLE
[FSPE-7338] Added delay to sidenav with openOnHover

### PR DESCRIPTION
**Pull request by janibolkvadze**
https://ctl.atlassian.net/browse/FSPE-7338

### CODE REVIEW

**Comment by allyourbotsarebelongtous** [comment]
Your CHI Pull Request website is available for review at <a href="https://7dd5f0523d1900c407645731a1604ff930d0d0d8-chi-pr.enterprise.ctl.io" target="_blank"> https://7dd5f0523d1900c407645731a1604ff930d0d0d8-chi-pr.enterprise.ctl.io </a>

**Comment by allyourbotsarebelongtous** [comment]
Build finished SUCCESSFULLY. See http://jenkins.elasticbox.com/job/chi_pull_request/1786/ for more details.

**Comment by allyourbotsarebelongtous** [comment]
Your CHI Pull Request website is available for review at <a href="https://cdaee134150ae2890b4a62b1b0420854abbd6351-chi-pr.enterprise.ctl.io" target="_blank"> https://cdaee134150ae2890b4a62b1b0420854abbd6351-chi-pr.enterprise.ctl.io </a>

**Comment by allyourbotsarebelongtous** [comment]
Build finished SUCCESSFULLY. See http://jenkins.elasticbox.com/job/chi_pull_request/1804/ for more details.

**Comment by allyourbotsarebelongtous** [comment]
Your CHI Pull Request website is available for review at <a href="https://3aad6e89887bb04d15b47f6622b79189b05eb851-chi-pr.enterprise.ctl.io" target="_blank"> https://3aad6e89887bb04d15b47f6622b79189b05eb851-chi-pr.enterprise.ctl.io </a>

**Comment by allyourbotsarebelongtous** [comment]
Build finished SUCCESSFULLY. See http://jenkins.elasticbox.com/job/chi_pull_request/1807/ for more details.

**Review by jllr — APPROVED** [review]

File: src/chi/javascript/components/sidenav.js [hunk 1]
```diff
@@ -15,15 +15,14 @@ const DRAWER_LINKLIST_CLASS = "chi-sidenav__drawer-list";
 const DRAWER_ITEM_LIST_CLASS = "chi-sidenav__drawer-item-list";
 const DRAWER_ITEM_TAB_CLASS = "chi-sidenav__drawer-item-tab";
 const DRAWER_SUBITEM_TRIGGER_CLASS = "chi-drawer__subitem-list-trigger";
-
+const OPEN_ON_HOVER_DELAY = 300;
```
Comment (mattnickles) [inline src/chi/javascript/components/sidenav.js:18]: 👍 300 is looking much better

**Review by mattnickles — APPROVED** [review]

**Review by cristb — APPROVED** [review]

**Comment by allyourbotsarebelongtous** [comment]
The following instances are being terminated: https://ops.elasticbox.com/services/instances/i-o45f0c, https://ops.elasticbox.com/services/instances/i-ce01wu, https://ops.elasticbox.com/services/instances/i-huqiu2

**Pull request merged**